### ISSUE TITLE
Files with .obj extension now have problems when working with rgl function in Shiny App before

### ISSUE THREAD

**Issue by kahvecirem** [dmurdoch/rgl#257]

**Comment by dmurdoch** [comment]
Could you please provide some details?

**Comment by kahvecirem** [comment]
Sorry for my misspelling. I forgot to revise. I have a file with .obj extension, but when I scanned it two weeks ago, there was no problem, now when I scan it on shiny app, it does not output my .obj file. But the same code works in normal R interface thanks to rgl.  I tried getting solution in many places including Stackoverflow but it didn't work. 

My code simply looks like this: 

open3d(),
shade3d(readOBJ("tirtill.obj", material = list(color = "green",  shininess = 1)))
